nginx.conf

```diff
@@ -0,0 +1,25 @@
+server {
+	listen 80 default_server;
+
+	gzip on;
+	gzip_min_length 1000;
+	gzip_types text/plain text/xml application/javascript text/css;
+
+	root /app;
+
+	# normal routes
+	# serve given url and default to index.html if not found
```

Comment:
Shouldn't it return a 404 page if a wrong link has been entered?

Reply:
SPA handles all routes, nginx should not handle any.